action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: 'Policy for outbound traffic, can be either audit or block'
     required: false
     default: 'block'
+  expected_checksum:
+    description: 'Expected sha256 checksum of latest agent.tar.gz file'
+    default: "a5f466fc5c8a9b809afd421e0f32903da98908feab5a245c734d3775e2e10032"
```

Comment:
@h0x0er the expected checksum should not be controllable by the user of the Action. You will need to hardcode this value in the checksum_verify.ts file.

Reply:
@varunsh-coder I removed the checksum from action-input and hard-coded it in `checksum.ts`